library/ziti.c

```diff
@@ -712,12 +726,20 @@ static void session_cb(ziti_session *session, ziti_error *err, void *ctx) {
         free_ziti_session(old_session);
         FREE(old_session);
 
+        uv_timeval64_t now;
+        uv_gettimeofday(&now);
+
+        int time_diff = (int) (now.tv_sec - session->updated->tv_sec);
+        if (abs(time_diff) > 10) {
```

Comment:
why not > 0? why the leeway?

Reply:
it is just an informational error message, we have no control of how often host time is synced with UTC, giving it leeway avoids logging this too often